osudroidresource/src/main/res/values-pt-rBR/options.xml

```diff
@@ -6,59 +6,223 @@
+
     <string name="opt_special">Especial</string>
-    <string name="opt_deleteosz_title">Apagar arquivos .osz</string>
-    <string name="opt_deleteosz_summary">Apaga os arquivos .osz após serem importados</string>
-    <string name="opt_directory_title">Pasta dos beatmaps</string>
-    <string name="opt_directory_summary">Caminho para a pasta de beatmaps. (por padrão: /mnt/sdcard/osu!droid/Songs)</string>
-    <string name="opt_clear_title">Remover biblioteca em cache</string>
-    <string name="opt_clear_summary">Força a remoção da biblioteca de beatmaps salva em cache (Isto não ira remover seus beatmaps).</string>
-    <string name="opt_doublesound_title">Sons duplos nos clicks</string>
-    <string name="opt_doublesound_summary">Faz os sons se parecerem mais com os do osu! original!</string>
+    <string name="opt_taiko_title">Carregar sons do T-Aiko!</string>
+    <string name="opt_taiko_summary">Também carregará sons do diretório do T-Aiko!</string>
+
+    <string name="opt_deleteosz_title">Deletar arquivos .osz</string>
+    <string name="opt_deleteosz_summary">Deletar arquivos .osz após importa-los</string>
+
+    <string name="opt_directory_title">Caminho dos beatmaps</string>
+    <string name="opt_directory_summary">Caminho do diretório de beatmaps (padrão: /mnt/sdcard/osu!droid/Songs)</string>
+
+    <string name="opt_clear_title">Cache da biblioteca</string>
+    <string name="opt_clear_summary">Força a limpeza do cache da biblióteca</string>
+
+    <string name="opt_doublesound_title">Hitsounds melhores</string>
+    <string name="opt_doublesound_summary">Faz com que os sons fiquem mais semelhantes ao do osu! original</string>
+
     <string name="opt_sound">Som</string>
-    <string name="opt_fps_title">Mostrar FPS</string>
-    <string name="opt_fps_summary">Mostra taxa de quadros por segundos (FPS) durante o jogo.</string>
-	<string name="opt_lowtextures_title">Reduzir qualidade das texturas.</string>
-    <string name="opt_lowtextures_summary">Reduz a qualidade das texturas do jogo. Se desativado, o jogo passara a utilizar as texturas de alta resolução (2x).</string>
-    <string name="opt_nativeplayer_title">Utilizar o reprodutor integrado</string>
-    <string name="opt_nativeplayer_summary">Utilizar o reprodutor integrado para maior precisão (arquivos .ogg não suportados).</string>
-    <string name="opt_bgbrightness_title">Brilho do fundo</string>
-    <string name="opt_bgbrightness_summary">Altera o brilho de fundo</string>
+
+    <string name="opt_fps_title">Mostrar o FPS</string>
+    <string name="opt_fps_summary">Mostra o FPS (Frames Por Segundo) durante a gameplay</string>
+
+    <string name="opt_skipoffset_title">Offset de skip</string>
+    <string name="opt_skipoffset_summary">Esse offset vai ser adicionado quando você pressionar o botão de &quot;Skip&quot;. Coloque -1 para um offset automático</string>
```

Comment:
remove 'de' from the ``pressionar o botão`` part and substitute ``Coloque`` to ``Digite``

Reply:
did it